_data/sites.json

```diff
@@ -7850,17 +7860,12 @@
 
     {
         "name": "Hacker News",
-        "url": "https://jacquesmattheij.com/the-unofficial-hn-faq/#deleteaccount",
-        "difficulty": "impossible",
-        "notes": "Your contributions are there to stay, but you can at least clear out your profile -- even your email address.",
-        "notes_tr": "Katkılarınız kalıcıdır, ancak en azından profilinizi, hatta e-posta adresinizi bile temizleyebilirsiniz.",
-        "notes_fr": "Vos contributions resteront sur le site, mais vous pouvez 'vider' votre profil, même votre email.",
-        "notes_it": "Le tue donazioni resteranno, ma puoi almeno pulire il tuo profilo -- anche il tuo indirizzo email",
-        "notes_pt_br": "Suas contribuições serão mantidas, mas você pode limpar o seu perfil -- inclusive seu endereço de e-mail.",
-        "notes_cat": "Les seves contribucions es mantindran, però pot netejar el seu perfil -- incloent el seu e-mail.",
-        "notes_es": "Tus contribuciones se permanecerán, pero puedes borrar los datos en tu perfil (incluso tu e-mail).",
+        "url": "https://news.ycombinator.com/item?id=23623799",
+        "difficulty": "hard",
+        "email": "hn@ycombinator.com",
+        "notes": "You can remove your account's data, includng your emaill, but removal of your posts should be negotiated with the staff, as it may harm the project.",
```

Comment:
```suggestion
        "notes": "You can remove your account's data, including your email, but removal of your posts should be negotiated with the staff, as it may harm the project.",
```